math/Frame3f.cs

```diff
@@ -68,6 +69,12 @@ public Frame3f(Vector3f origin, Vector3f x, Vector3f y, Vector3f z)
             this.rotation = m.ToQuaternion();
         }
 
+        public Frame3f(double x, double y, double z, double rx, double ry, double rz, double rw)
```

Comment:
Since you cast double to float either way, wouldn't it be easier to just accept floats in constructor?

Reply:
Sadly, I usually use double, and, then, to call it, I would need to write `new Frame3f((float)0.5, ...)` or `new Frame3f(0.5f, ...)`. Both are waste of time